src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --blue-color: #00acdc;
+}
+
+.header {
+  width: 100%;
+  background-color: #fff;
+  display: flex;
+  flex-direction: row;
+  border-style: dotted;
+  padding: auto, 50px;
```

Comment:
The padding property is incorrectly formatted. Use spaces instead of commas to separate values, e.g., 'padding: 0 50px;'.